README.md

```diff
@@ -102,6 +122,13 @@ in lieu of command-line flags when using the above applications:
    reduces both latency and the number of Fleet API calls a client makes when
    reconnecting to a vehicle after restarting. This is particularly helpful
    when using `tesla-control`, which restarts on each invocation.
+ * `TESLA_TLS_CERT` specifies a TLS certificate file for the HTTP proxy.
+ * `TESLA_TLS_KEY` specifies a TLS key file for the HTTP proxy.
+ * `TESLA_HOST` specifies the host for the HTTP proxy to bind to.
+ * `TESLA_PORT` specifies the port for the HTTP proxy to bind to.
+ * `TESLA_VERBOSE` enables verbose logging for the HTTP proxy.
+ * `TESLA_TIMEOUT` specifies the timeout for the HTTP proxy to use when
```

Comment:
There are other timeouts used by the tools, so using a more specific name here could help avoid confusion over what timeout is being configured.